src/backend/marsha/core/models/account.py

```diff
@@ -20,11 +20,17 @@
 SHARED_SECRET_CHARS = string.ascii_letters + string.digits + "!#$%&*+-=?@^_"
 SHARED_SECRET_SIZE = 40
 
-ADMINISTRATOR, INSTRUCTOR, STUDENT = ("administrator", "instructor", "student")
+ADMINISTRATOR, INSTRUCTOR, STUDENT, NONE = (
+    "administrator",
+    "instructor",
+    "student",
+    "none",
+)
 LTI_ROLES = {
     ADMINISTRATOR: {"administrator"},
     INSTRUCTOR: {"instructor", "teacher", "staff"},
     STUDENT: {"student", "learner"},
+    NONE: {"none"},
```

Comment:
I found the idea of "anonymous" user clearer and in line with what Django does?

Reply:
In the spec `anonymous` does not exists. This is why I choose `NONE` https://www.imsglobal.org/specs/ltiv1p0/implementation-guide#toc-6